docs/export_results.rst

```diff
@@ -0,0 +1,166 @@
+Export your results
+===================
+Taichi has functions that help you **export visual results to images or videos**. This tutorial demonstrates how to use them step by step.
+
+Export images
+-------------
+
+- There are two ways to export visual results of your program to images.
+- The first and easier way is to make use of ``ti.GUI``.
+- The second way is to call some Taichi functions such as ``ti.imwrite``.
+
+Export images using ``ti.GUI.show``
++++++++++++++++++++++++++++++++++++
+
+- ``ti.GUI.show(filename)`` can not only display the GUI canvas on your screen, but also save the image to your specified ``filename``.
+- Note that the format of the image is fully determined by the suffix of ``filename``.
+- Taichi now supports saving to ``png``, ``jpg``, and ``bmp`` formats.
+- We recommend using ``png`` format. For example:
+
+.. code-block:: python
+
+    import taichi as ti
+    import os
+
+    ti.init()
+
+    pixels = ti.var(ti.u8, shape=(512, 512, 3))
```

Comment:
Should we inform that all `ti.f32, ti.f64, ti.u8, ti.u16, ti.u32, ti.u64` is available? People like to store image in `ti.f32` very much, which is supported by most platforms.